awx/main/dispatch/worker/base.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, name, worker, queues=[], pool=None):
             self.pool = WorkerPool()
         self.pool.init_workers(self.worker.work_loop)
         self.redis = redis.Redis.from_url(settings.BROKER_URL)
+        self.redis.client_setname("awx_consumer_base")
```

Comment:
Could we use `self.redis.client_setname(name)` instead?  This value would change depending on if it's the dispatcher or callback receiver.

Reply:
This is going to be either `"callback_receiver"` or `"dispatcher"` if `name` is used.